packages/app/package.json

```diff
@@ -76,7 +76,8 @@
     "ios": {
       "firebase": "11.4.0",
       "iosTarget": "13.0",
-      "macosTarget": "10.15"
+      "macosTarget": "10.15",
+      "tvosTarget": "12.0"
```

Comment:
This should be 13.4 or higher for compatibility with RNTV 0.73 and later

Reply:
Bit of a catch :) If we use firebase_ios_target for tvOS then there isn't a way set tvosTarget to to 13.4 without also setting iosTarget, which I'm not sure we want to do when most users aren't targeting tvOS?

Reply:
I'm okay with having these separate, and we align the minimums with the firebase-ios-sdk podspecs not with react-native. The Auth podspec over there is the highest one so it determines all, and it is currently at 13.0:

https://github.com/firebase/firebase-ios-sdk/blob/ae2554220afb03a5d6fc389c28a86061a12c7caa/FirebaseAuth.podspec#L22-L25

So in final form what I will do here is keep them separate, but bump to 13.0